travis/manylinux1_x86_64-glpk/build_lpsolve.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+LPS_VER="5.5.2.5"
+
+# Must download this manually due to old OpenSSL
+# wget https://sourceforge.net/projects/lpsolve/files/lpsolve/${LPS_VER}/lp_solve_${LPS_VER}_source.tar.gz
+
+# TODO add checksum
```

Comment:
TODO

Reply:
This will get complete once manylinux2010 is relased and wget is updated.